public_officials/functional_tests/test_public_officials.py

```diff
@@ -47,9 +47,11 @@ def test_checks_for_content_on_official_show(self):
             self.browser.get(self.live_server_url + '/legislators/%d/' % legislator.id)
             name_text = self.browser.find_element_by_tag_name('h1').text
             status_text = self.browser.find_element_by_tag_name('h3').text
+            follow_button_text = self.browser.find_element_by_css_selector('.follow').text
             bio_text = self.browser.find_element_by_css_selector('.profile').text
             stats_text = self.browser.find_element_by_css_selector('.stats').text
             self.assertIn("Diana DeGette", name_text)
+            self.assertIn("Follow", follow_button_text)
```

Comment:
Looks great. What do we think we'll want to do with the button for an unauthenticated user? Deactivate it or just not display it?